source/features/add-co-authored-by.js

```diff
@@ -0,0 +1,125 @@
+import select from 'select-dom';
+import * as api from '../libs/api';
+import {escapeForGql} from '../libs/utils';
+import {getOwnerAndRepo, getPRNumber} from '../libs/page-detect';
+
+const coAuthorData = {};
+
+async function fetchCoAuthoredData() {
+	const {ownerName, repoName} = getOwnerAndRepo();
+	const prNumber = getPRNumber();
+
+	if (!ownerName || !repoName || !prNumber) {
+		return;
+	}
+
+	const {data: contributorData} = await api.v4(
+		`{
+			repository(owner: "${ownerName}", name: "${repoName}") {
+				pullRequest(number: ${prNumber}) {
+					commits(first: 100) {
+						nodes {
+							commit {
+								author {
+									user {
+										databaseId
+										login
+										name
+										email
+									}
+								}
+							}
+						}
+					}
+					reviews(first: 100) {
```

Comment:
`pullRequest` has a `participants` field that lets you query emails directly:

        participants(first: 100) {
          nodes {
            login
            databaseId
            email
          }
        }

<img width="349" alt="" src="https://user-images.githubusercontent.com/1402241/50326276-14efe500-051c-11e9-902b-c6eb7a75c8f3.png">

Reply:
It looks like `participants` includes requested reviewers who _haven't_ left a review, which I don't think we really need to include. 🙂